imageio/plugins/tifffile.py

```diff
@@ -191,6 +191,8 @@
     "resolution",
     "description",
     "compress",
+    "compression",
```

Comment:
Is that all? Maybe we can add the entire set of new args they currently have?
- https://github.com/cgohlke/tifffile/blob/a73812e98f58b068a590de862ed547820fd332ad/tifffile/tifffile.py#L1514-L1551

Reply:
We could; although we would also have to also document those new kwargs. I would much rather spend that time on writing a v3 plugin for tifffile. That would also solve the problem of reoccurring maintenance costs for keeping the plugin in sync.

Speaking off documentation, I just noticed that I forgot to add documentation for `compression` and `compressionargs` ... will do :)

Reply:
IMHO undocumented feature is still better than not working, so if I'm doing it I would probably just copy-paste their arg list here. But anyway, my issue is resolved so will let someone else to worry about the rest.